cmd/lakectl/cmd/common_helpers.go

```diff
@@ -296,10 +296,18 @@ func MustParseRepoURI(name, s string) *uri.URI {
 }
 
 func MustParseRefURI(name, s string) *uri.URI {
+
+	lastChar := s[len(s)-1:]
+	if lastChar == "/" {
+		s = strings.TrimSuffix(s, "/")
+		fmt.Printf("INFO: %s/ is not a valid URI, showing a diff for %s instead\n", s, s)
```

Comment:
`MustParseRefURI` may be called for almost any lakectl command, not merely diff.  This message might use `name`, or it might simply use a format string like
```suggestion
		fmt.Printf("INFO: %s/ is not a valid URI, using %s instead\n", s, s)
```